splitgraph/ingestion/sqlite/__init__.py

```diff
@@ -245,11 +266,18 @@ def _batched_copy(
             insert_table_contents = (
                 table_contents if len(primary_keys) > 0 else [row[1:] for row in table_contents]
             )
-            self.engine.run_sql_batch(
-                SQL("INSERT INTO {0}.{1} ").format(Identifier(schema), Identifier(table_name))
-                + SQL(" VALUES (" + ",".join(itertools.repeat("%s", len(schema_spec))) + ")"),
-                insert_table_contents,
-            )  # nosec
+            try:
+                self.engine.run_sql_batch(
+                    SQL("INSERT INTO {0}.{1} ").format(Identifier(schema), Identifier(table_name))
+                    + SQL(" VALUES (" + ",".join(itertools.repeat("%s", len(schema_spec))) + ")"),
+                    insert_table_contents,
+                )  # nosec
+            except Exception as e:
+                print(
+                    "Received exception %s running query '%s' with parameters %s"
+                    % (str(e), query, str(parameters))
+                )
+                raise e
```

Comment:
`logging.error` would work better here and emit the stacktrace. Something like:

```suggestion
                logging.error(
                    "Error running query '%s' with parameters %s",
                    query, str(parameters),
                    exc_info=e
                )
                raise
```